recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -56,6 +55,8 @@ IMAGE_INSTALL += " \
 
 do_image_zip() {
     cd ${DEPLOY_DIR_IMAGE}/
-    sha256sum opentrons-ot3-image-verdin-imx8mm.ext4.xz > rootfs.xz.256
-    zip ot3-system.zip opentrons-ot3-image-verdin-imx8mm.ext4.xz rootfs.xz.256
+    /bin/pwd
+    ls -l
+    sha256sum ${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.ext4.xz > rootfs.xz.256
+    zip ot3-system.zip ${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.ext4.xz rootfs.xz.256
```

Comment:
at some point we'll probably want something that lets us set what goes into the zipfile using oe variables or something so recipes can do like `OPENTRONS_SYSTEM_UPDATE_append = " some-random-file"` but that doesn't have to be here